x/emissions/keeper/queryserver/query_server_inferences.go

```diff
@@ -231,7 +226,7 @@ func (qs queryServer) GetConfidenceIntervalsForInferenceData(
 	for _, forecast := range networkInferences.ForecasterValues {
 		weight, exists := forecasterWeights[forecast.Worker]
 		if exists {
-			inferences = append(inferences, forecastImpliedInferenceByWorker[forecast.Worker].Value)
+			inferences = append(inferences, forecast.Value)
```

Comment:
In the place where we used to have a nil `*Inference`, we now will not have that - but may have an empty `Dec`. Is that an OK solution?

Reply:
yeah, since `forecastImpliedInferenceByWorker` was exactly `networkInferences.ForecasterValues` converted to map[string]*Inference. I don't think we have any problems here.

Reply:
What I mean here is that we sometimes had a `nil`  `*Inference`, right? 
Now, what would be the expected behaviour for the case that previously generated that nil `*Inference` ? 
How are we expecting it to behave in that case now?

Does that generating a no-entry on `ForecastValues` ? 
Or an entry with an empty `Dec`? 
And how is that handled downstream?

Reply:
I don't think we could have a nil inference here, actually. What is in ForecasterValues is exactly the existing forecast-implied inferences, we don't do any fallback for an empty value.